### PR TITLE
Added delete functionality to home page 

### PR DESCRIPTION
**Pull request by OlypsisAli**
Previously users could not delete files uploaded to estuary via the dashboard. They now have this ability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Now view [localhost:4444](http://localhost:4444) in your browser.
 
 **Note:** If you are using another port for your estuary node other than the default port (`localhost:3004`), you can set the `ESTUARY_API` environment variable to point to the right URL. for example. to change to port 9999
 ```bash
-$ ESTUARY_API=http://localhost:9999 npm run dev
+$ c
```
Comment (jimmylee) [inline README.md:20]: Wait whats this?

File: components/FilesTable.tsx [hunk 2]
```diff
@@ -52,11 +56,40 @@ const FilesTable = ({ files }) => {
...
         disableFilters: true,
       },
+      {
+        Header: 'Delete',
+        accessor: (data) => String(data.id).padStart(9, '0'),
+        Cell: ({ value }) => (
+          <div style={{ fontSize: 12, fontFamily: 'Mono' }}>
+            {' '}
+            <Button
+              htmlFor="FILE_UPLOAD_TARGET"
```
Comment (jimmylee) [inline components/FilesTable.tsx:66]: I think this ` htmlFor` should be removed, it doesn't belong here

**Review by jimmylee — CHANGES_REQUESTED** [review]
Some minor nits, also make sure you have `prettier`, thank you jumping in like this @OlypsisAli

**Comment by OlypsisAli** [comment]
Forgot to add prettier. Will do so when I step back in.

**Comment by jimmylee** [comment]
Needs to fix conflict

**Comment by jimmylee** [comment]
Closing this until you have a chance to fix conflicts

**Comment by OlypsisAli** [comment]
@jimmylee Was on a family vacation for the past few weeks. Will get this fixed up.

**Pull request closed without merging**